addons/mail/static/src/core/common/discuss.scss

```diff
@@ -17,27 +21,27 @@
 .o-mail-Discuss-threadName {
     max-width: 75%;
 }
-
 .o-mail-Discuss-header {
     height: $o-mail-Discuss-headerHeight;
+    box-shadow: 0px 1px 6px -3px rgba(50, 50, 50, 0.15);
+}
 
-    button {
-        &:not(.o-isActive):hover {
-            background-color: $gray-200;
-        }
-        &.o-isActive {
-            background-color: $gray-300;
-        }
-        &.o-isActive:hover {
-            background-color: $gray-300;
-        }
+.o-mail-Discuss-headerActions button {
+    &:not(.o-isActive):hover {
+        background-color: $gray-200;
+    }
+    &.o-isActive {
+        background-color: $gray-300;
+    }
+    &.o-isActive:hover {
+        background-color: $gray-300;
     }
 }
 
 .o-mail-Discuss-threadAvatar {
     img {
-        height: 36px;
-        width: 36px;
+        height: 32px;
+        width: 32px;
```

Comment:
possible to use variable?

Reply:
No variable with this sizing of avatar.
We have:
```scss
$o-mail-Avatar-size: 36px !default;
$o-mail-Avatar-sizeSmall: 24px !default;
```
36px is too big, 24px too small.

I think this idea to only have a few selected sizes of images is too constraining when the actual good fit is somewhere else. Other messaging apps use many contextual sizes for avatar, so I think we should stop blindly harmonise when this is not the right way like here.